lib/src/main/java/org/team100/lib/commands/drivetrain/DriveWithProfile.java

```diff
@@ -66,36 +123,40 @@ public void initialize100() {
     @Override
     public void execute100(double dt) {
         if (m_fieldRelativeGoal.get() != null) {
-        Rotation2d currentRotation = m_swerve.getPose().getRotation();
-        // take the short path
-        double measurement = currentRotation.getRadians();
-        Rotation2d bearing = new Rotation2d(
-                Math100.getMinDistance(measurement, m_fieldRelativeGoal.get().getRotation().getRadians()));
+            Rotation2d currentRotation = m_swerve.getPose().getRotation();
+            // take the short path
+            double measurement = currentRotation.getRadians();
+            Rotation2d bearing = new Rotation2d(
+                    Math100.getMinDistance(measurement, m_fieldRelativeGoal.get().getRotation().getRadians()));
 
-        // make sure the setpoint uses the modulus close to the measurement.
-        thetaSetpoint = new State100(
-                Math100.getMinDistance(measurement, thetaSetpoint.x()),
-                thetaSetpoint.v());
-        State100 thetaGoal = new State100(bearing.getRadians(), 0);
-        State100 xGoalRaw = new State100(m_fieldRelativeGoal.get().getX(),0,0);
-        xSetpoint = xProfile.calculate(0.02, xSetpoint, xGoalRaw);
-        State100 yGoalRaw = new State100(m_fieldRelativeGoal.get().getY(),0,0);
-        ySetpoint = yProfile.calculate(0.02, ySetpoint, yGoalRaw);
-        // State100 thetaGoalRaw = new State100(m_robotRelativeGoal.get().getRotation().getRadians(),0,0);
-        thetaSetpoint = thetaProfile.calculate(0.02, thetaSetpoint, thetaGoal);
-        SwerveState goalState = new SwerveState(xSetpoint, ySetpoint, thetaSetpoint);
-        Twist2d goal = m_controller.calculate(m_swerve.getState(), goalState);
-        m_swerve.driveInFieldCoords(goal, 0.02);
+            // make sure the setpoint uses the modulus close to the measurement.
+            thetaSetpoint = new State100(
+                    Math100.getMinDistance(measurement, thetaSetpoint.x()),
+                    thetaSetpoint.v());
+            State100 thetaGoal = new State100(bearing.getRadians(), 0);
+            State100 xGoalRaw = new State100(m_fieldRelativeGoal.get().getX(), 0, 0);
+            xSetpoint = xProfile.calculate(0.02, xSetpoint, xGoalRaw);
```

Comment:
replace all these instances of "0.02" with "dt" since it's a better estimate of the duration of the following period